2020-01-Winter/4_trees/README.md

```diff
@@ -23,21 +23,56 @@ Problems and solutions for Trees session on February 14, 2020.
 
 <a name="p1"/>
 
-### 1. PROBLEM 1 TODO :bug:
+### 1. Path Sum
 
-Source: TODO :bug:
+Source: [Leetcode] https://leetcode.com/problems/path-sum/
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Problem Statement: Given a binary tree and a sum, determine if the tree has a root-to-leaf path such that adding up all the values along the path equals the given sum.
 
-#### Example Input
+Note: A leaf is a node with no children.
 
-If the problem is simple enough, remove this section. TODO :bug:
+#### Example Input
+Example 1:
+
+Given the below binary tree and sum = 22,
+
+      5
+     / \
+    4   8
+   /   / \
+  11  13  4
+ /  \      \
+7    2      1
+return true, as there exist a root-to-leaf path 5->4->11->2 which sum is 22.
+
+Example 2:
+Given the below binary tree and sum = 1,
+      1
+     / \
+    -2  3
+return false, as there does not exist a root-to-leaf path which sums 1.
 
 #### Function Signature
-
-TODO :bug:
+Example C++ function signature:
+
+/**
```

Comment:
Without enclosing code blocks in triple backticks, you will wind up with some very strange rendering on GitHub's part. This happens because the code-specific characters, indentation, etc are getting recognized as Markdown formatting instead of a code block. Here's what it looks like before the changes:

![code_block](https://user-images.githubusercontent.com/29528527/74423510-448b3c00-4e05-11ea-81d1-f1a070fed561.png)


````suggestion
```c++
/**
````